react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -198,7 +233,7 @@ class ConnectionStatsTable extends Component<Props> {
      */
     render() {
         const { isLocalVideo, enableSaveLogs, disableShowMoreStats, classes } = this.props;
-        const className = isMobileBrowser() ? 'connection-info connection-info__mobile' : 'connection-info';
+        const className = clsx(classes.connectionStatsTable, { [classes.mobile]: isMobileBrowser() });
```

Comment:
Please change the object declaration to `&&` so that it's more in line with the rest of the app, and also easier to understand for those that aren't familiar with clsx syntax

Reply:
Using `&&` would actually defat the purpose of `clsx` package. As I mentioned it is a very simple and small library that most developers use every day and if someone doesnt know what it is, they can learn about it in a minute.